pkg/awsclient/client.go

```diff
@@ -354,8 +356,8 @@ func (c *awsClient) ChangeResourceRecordSets(input *route53.ChangeResourceRecord
 	return c.route53client.ChangeResourceRecordSets(input)
 }
 
-// NewClient creates our client wrapper object for the actual AWS clients we use.
-func NewClient(awsAccessID, awsAccessSecret, token, region string) (Client, error) {
+// newClient creates our client wrapper object for the actual AWS clients we use.
+func newClient(awsAccessID, awsAccessSecret, token, region string) (Client, error) {
```

Comment:
Renamed to make private, because only GetClient (below) should be using this directly.